auth/user-profile/android.md

```diff
@@ -45,14 +46,34 @@ publicDB.save(user, new RecordSaveResponseHandler() {
     }
 })
 ```
+```kotlin
```

Comment:
Please add `onPartiallySaveSuccess` in both version.

Reply:
No, I don't think the intention is to create a new record. If we are going to create a new user, then signup is being called.
Please take a look at the below code
```java
Record currentUser = this.skygear.getAuth().getCurrentUser();
currentUser.set("gender", "male");
this.skygear.getPublicDatabase().save(currentUser,  ...);
```